src/webprojects/sumo.md

```diff
@@ -45,7 +45,7 @@ Localization updates are pushed to the staging and production environments usual
 
 ## FAQ
 
-For any additional questions not covered here, you can contact [Rubén](mailto:nukeador@mozilla.com).
+For any additional questions not covered here, you can reach out to the #sumo room in [Matrix](https://chat.mozilla.org/#/room/#sumo:mozilla.org0) to discuss it.
```

Comment:
I assume the 0 at the end of the URL it not needed

Reply:
Oops, let me fix it.